decidim-admin/config/locales/en.yml

```
@@ -136,25 +136,25 @@ en:
       actions:
         add: Add
         attachment:
-          new: New %{name}
+          new: New attachment
         attachment_collection:
-          new: New %{name}
+          new: New attachment_collection
```

Since this is aimed to the user, I'd remove the `_`:

```suggestion
          new: New attachment collection
```

This will be easier to read to end users 😄